### DISCUSSION TITLE
Auto Reply tidak berfungsi

### DISCUSSION THREAD

**Discussion by baroescoeg** [aldinokemal/go-whatsapp-web-multidevice#146] [General]
Saya sudah mengikuti semua tutorial
Semua berjalan dan berfungsi sebagaimana mesti nya
tapi untuk **Auto Reply** tidak berfungsi
Bagaimana cara mengatasi nya ?

Terimakasih

**Comment by aldinokemal** [comment]
![picture](https://github.com/aldinokemal/go-whatsapp-web-multidevice/assets/14232125/ff2b3045-ce93-413a-91fc-8b3e4067d9d6)
___
Ini sudah saya coba tidak ada masalah, bisa disertakan detail step yang telah dilakukan @baroescoeg ?

**Comment by ayenz** [comment]
@aldinokemal Hallo saya juga ada sedikit kendala di sini. Auto reply tidak berjalan ketika dibalas menggunakan whatsapp windows / macos, namun kalau kita balas menggunakan aplikasi smartphone works well
![Screenshot 2024-07-24 092849](https://github.com/user-attachments/assets/be41d75f-4bce-4cd4-ba2d-735411f9c574)
Yang saya cheklist merah, saya reply menggunakan whatsapp app for windows.
